tests/test_privateca.py

```diff
@@ -96,18 +96,20 @@ def test_privateca_daemonset_enabled_with_custom_image(self, kube_version):
         assert len(docs) == 1
         doc = docs[0]
         self.common_tests_daemonset(doc)
-        assert any(
-            "snarks:boojums" in item
-            for item in jmespath.search("spec.template.spec.containers[*].image", doc)
+        assert len(docs[0]["spec"]["template"]["spec"]["containers"]) == 1
+        assert (
+            doc["spec"]["template"]["spec"]["containers"][0]["image"]
+            == "snarks:boojums"
         )
 
 
 @pytest.mark.parametrize(
     "kube_version",
     supported_k8s_versions,
 )
-class TestPSP:
+class TestPrivateCaPsp:
```

Comment:
```suggestion
class TestPrivateCApsp:
```

Reply:
Wouldn't `test private CA psp` be TestPrivateCaPsp? That's what https://textedit.tools/camelcase shows.

Reply:
Typically you don't mix camelCase in abbreviations: good example

HTTPHeader instead of HttpHeader.


More examples https://stackoverflow.com/questions/15526107/acronyms-in-camelcase

It's not blocking review just FYI.